### ISSUE TITLE
Windows high contrast mode

### ISSUE THREAD

**Issue by jack-skerrett-oak** [oaknational/Oak-Web-Application#765]
There are some oddities happening when you view the site in high contrast mode on windows.
- LinkedIn icon in footer and nav menu does not behave the same as other icons
- Manage cookie settings does not act like other footer elements
- In some stylings, some elements are not visible at all, though I am unsure how much of this we can actually control 

![Image](https://user-images.githubusercontent.com/102532470/191748504-424c5c62-3033-49bc-859b-719f7ce0c819.png)
![Image](https://user-images.githubusercontent.com/102532470/191748508-325a24f1-227c-49d6-83a7-d33b0efee676.png)
![Image](https://user-images.githubusercontent.com/102532470/191748506-d10cbc32-fde0-4e50-a2f4-bed44ddb9680.png)
![Image](https://user-images.githubusercontent.com/102532470/191748505-086f3bb5-cb2d-4e83-b349-df1acf404de5.png)
![Image](https://user-images.githubusercontent.com/102532470/191748507-03587682-290f-4b58-ac13-3c8401c46d43.png)
